resources/user-agents/email-clients/eudora/eudora.json

```diff
@@ -0,0 +1,56 @@
+{
+  "division": "Eudora",
+  "sortIndex": 1670,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Eudora",
+      "platform": "OSX",
+      "engine": "WebKit",
+      "device": "Macintosh",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Eudora",
+        "Browser": "Eudora",
+        "Browser_Type": "Email Client",
+        "Browser_Maker": "Qualcomm",
+        "Alpha": "false",
+        "Beta": "false",
+        "isSyndicationReader": "false",
+        "Crawler": "false"
+      },
+      "children": [
+        {
+          "match": "Eudora/* (MacOS)"
+        }
+      ]
+    },
+    {
+      "userAgent": "Eudora OSE",
+      "platform": "Windows",
+      "engine": "Gecko",
+      "device": "Windows Desktop",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Eudora OSE",
+        "Browser": "Eudora OSE",
```

Comment:
What is "OSE"?

Reply:
Open Source Edition

Reply:
Shall I use the full name?

Reply:
Ah, maybe in the Comment?

Also, it seems that perhaps Eudora OSE also ran/runs on OSX and Linux based on this wiki, though I can't find any example UAs in our logs.  https://wiki.mozilla.org/Eudora_OSE